src/openms/source/FORMAT/MzTab.cpp

```diff
@@ -2686,11 +2703,54 @@ Not sure how to handle these:
     Size psm_search_engine_index(1);
     for (auto const & se : search_engine_to_runs) // loop over (unique) search engine names
     {
+      //Huge TODO: we need to somehow correctly support peptide-level scores
       MzTabParameter psm_score_type;
+      MzTabParameter pep_score_type;
       const tuple<String, String, String>& name_version_score = se.first;
-      psm_score_type.fromCellString("[,," + get<0>(name_version_score) + " " + get<2>(name_version_score) + "," + get<1>(name_version_score) + "]");
+
+      //TODO we also have to consider the different q-value calculation methods of Percolator....
+      if (get<2>(name_version_score) == "peptide-level q-value")
```

Comment:
the problem I see is that the mzTab Peptide section is not really at the peptide level but basically just the consensusID level

Reply:
But what meaning does this value have in mztab?
The scores for the PEP section or scores for peptide sequence level?
I assume the first.

Reply:
yep first. though it is not 100% specified